config/printer-tevo-tarantula-pro-2020.cfg

```diff
@@ -75,13 +81,13 @@ pin_map: arduino
 kinematics: cartesian
 max_velocity: 400
 max_accel: 3000
-#max_accel_to_decel:
 max_z_velocity: 50
 max_z_accel: 100
-#square_corner_velocity: 5.0
 
 [heater_fan nozzle_fan]
 pin: ar7
+heater: extruder
+heater_temp: 50.0
```

Comment:
Is there a reason to explicitly state these?  The values are the defaults and it's generally preferable to not restate the defaults.

Reply:
Ah sorry, I didn't realise these were the defaults - makes sense not to re-define them!